### DISCUSSION TITLE
Check session on livewire method without page refresh

### DISCUSSION THREAD

**Discussion by Krato** [livewire/livewire#3781] [1. Help]
Hello and thanks for your help

I have a form with some steps without refreshing the page. Just a Livewire component with subcomponents.

I set the session expiration time to 5 minutes. And I want to check if the session has expired on every action on the form. If is expired then I will redirect to main form.

I can check the `session()` but is not giving me a good result cause the page is not being refreshed. Is there some way to test  if session has been expired without refreshing the page?

Thanks!

**Comment by joshhanley** [comment]
@Krato I don't have a good answer as I don't really know haha. 

But you should be able to check for a valid session using the normal Laravel methods, which will run even if the page isn't refreshed as the check should happen whenever an AJAX request is sent to Laravel/Livewire.

If I remember rightly, Livewire also a detection built in for expired session, but I think it just displays a browser message or something.

Hope this helps!
